### PR TITLE
Fix parseDevices() to enumerate multi-VPUs

### PR DESCRIPTION
**Pull request by rsreenu2000**
In Linux, MYRIAD M.2 form factor enumurates as "MYRIAD./dev/mxlk0:0".
This conflicts with the current implementation of parseDevices
which accounts only for MULTI or HETERO having a":" in devicename.
This fix allows to have multiple ":" in devcename and yet parse them correctly.

### CODE REVIEW

File: demos/common/samples/args_helper.hpp [hunk 1]
```diff
@@ -103,15 +103,18 @@ inline std::vector<std::string> split(const std::string &s, char delim) {
 }
 
 inline std::vector<std::string> parseDevices(const std::string& device_string) {
-    std::string comma_separated_devices = device_string;
-    const std::string::size_type colon_position = comma_separated_devices.find(":");
+    const std::string::size_type colon_position = device_string.find(":");
     if (colon_position != std::string::npos) {
-        comma_separated_devices = comma_separated_devices.substr(colon_position + 1);
+        std::string device_type = device_string.substr(0, colon_position);
+        if (device_type.compare("HETERO") == 0 || device_type.compare("MULTI") == 0) {
```
Comment (IRDonch) [inline demos/common/samples/args_helper.hpp]:
```suggestion
        if (device_type == "HETERO" || device_type == "MULTI") {
```

Comment (IRDonch) [reply demos/common/samples/args_helper.hpp]: @rsreenu2000 Ping?

Comment (rsreenu2000) [reply demos/common/samples/args_helper.hpp]: Are there any more open questions to be addressed?

Comment (IRDonch) [reply demos/common/samples/args_helper.hpp]: I have suggested a change. You can either accept it or explain why you disagree.

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**